### PR TITLE
Bump all packages below 1.0.0 to 1.0.0

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Bump all packages below 1.0.0 to 1.0.0

We keep getting restricted by starting at `0.1.0` as you are unable to make a clear distinction between minor and patch version bumps

**Code changes:**

- Bump all packages below 1.0.0 to 1.0.0
- Update contributing docs to reflect this new rule

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~[ ] Tests added for new features~
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-media-indicator/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-media-indicator",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "main": "dist/index.js",
```
Comment (sareh) [inline packages/components/psammead-media-indicator/package.json:4]: This package has not gone through accessibility review. We shouldn't update this to version 1.0.0 until we have done that.

File: packages/components/psammead-story-promo/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-story-promo",
-  "version": "0.1.2",
+  "version": "1.0.0",
   "main": "dist/index.js",
```
Comment (sareh) [inline packages/components/psammead-story-promo/package.json:4]: Same for this package.

File: packages/components/psammead-copyright/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
```
Comment (sareh) [inline packages/components/psammead-copyright/CHANGELOG.md:4]: You're missing `psammead-test-helpers` in this PR.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -95,7 +97,7 @@ The changelog should follow the following convention:
 
 | Version | Description |
 |---------|-------------|
-| 0.1.0 | [PR#49](https://github.com/bbc/psammead/pull/49) Initial creation of package |
+| 1.0.0 | [PR#49](https://github.com/bbc/psammead/pull/49) Initial creation of package |
```
Comment (sareh) [inline CONTRIBUTING.md:100]: Version 1.0.0 should be used once we've completed an accessibility swarm with our supported assistive tech and addressed changes necessary. https://bbc-news.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology
Likewise have checked browser compatibility against our support table. https://github.com/bbc/psammead/blob/latest/README.md#bar_chart-support-levels

**Comment by dr3** [comment]
Closing as not desired for now, will redo again eventually, simple change but the conflicts are crazy

**Pull request closed without merging**